### ISSUE TITLE
Request: Simulate all 4 character fights with a single click

### ISSUE THREAD

**Issue by Dannii-Darkness** [ed3ath/cbtracker#54]
Add an option to simulate and display fights for all 4 characters simultaneously! This would significantly increase the workflow for multi-window multi-accounts. 

Thanks. Love the tracker. Donating whatever I can afford.

**Comment by torzoman1986** [comment]
up

**Comment by Dannii-Darkness** [comment]
> up

ah sorry I think you misinterpreted what I meant - I should have phrased my request better.

I meant that it would simulate all YOUR 4 characters at the same time. ie. 16 simulated fights total simultaneously.

I made some mock-up JPG's just to make sure you get what I mean, which I have included.

Thanks
 
![cryptoblades1](https://user-images.githubusercontent.com/11830655/129355309-beff39c0-54a9-4daa-8e4b-56aba7e64082.jpg)
![cryptoblades2](https://user-images.githubusercontent.com/11830655/129355334-fba7e596-43bd-4b6c-a86a-be1c1f0227d8.jpg)
